_episodes/09-errors.md

```diff
@@ -65,7 +65,7 @@ IndexError                                Traceback (most recent call last)
 ----> 8 favorite_ice_cream()
 
 <ipython-input-1-70bd89baa4df> in favorite_ice_cream()
-      4         "vanilla",                                                                    "strawberry"
+      4         'vanilla',                                                                    'strawberry'
```

Comment:
Out of scope of this PR, but your change draws attention to the alignment/line numbers not being accurate in this error output. I will open an issue for this.